raylib/microui/microui_raylib_demo.odin

```diff
@@ -179,11 +179,13 @@ render :: proc "contextless" (ctx: ^mu.Context) {
 		switch cmd in variant {
 		case ^mu.Command_Text:
 			dst := rl.Rectangle{f32(cmd.pos.x), f32(cmd.pos.y), 0, 0}
-			for ch in cmd.str do if ch&0xc0 != 0x80 {
-				r := min(int(ch), 127)
-				src := mu.default_atlas[mu.DEFAULT_ATLAS_FONT + r]
-				render_texture(state.screen_texture, &dst, src, to_rl_color(cmd.color))
-				dst.x += dst.width
+			for ch in cmd.str {
```

Comment:
This is not related to this specific code but be should probably add support for the new `core:text/edit` integration to `vendor:microui`.

Reply:
I think it already is supported here, I can copy, paste etc.